electrum_nmc/electrum/names.py

```diff
@@ -145,6 +145,22 @@ def build_name_new(identifier: bytes, salt: bytes = None, address: str = None, p
 
     return {"op": OP_NAME_NEW, "commitment": commitment}, salt
 
+def validate_ipv4_address(address: str) -> None:
+    try:
+        ipaddress.IPv4Address(address)
+    except ipaddress.AddressValueError as e:
+        raise ValueError(f"Invalid IPv4 address: {e}")
+    except ipaddress.NetmaskValueError as e:
+        raise ValueError(f"Invalid IPv4 netmask: {e}")
+
+def validate_ipv6_address(address: str) -> None:
```

Comment:
This function should also verify that the string is in compressed form, not exploded.